docs/marionette.features.md

```diff
@@ -6,6 +6,14 @@ Marionette Features are opt-in functionality. That you can enable by setting `Ma
 + make it possible to add breaking changes in a minor release
 + give community members a chance to provide feedback for new functionality
 
+## `childViewEventPrefix`
+
+This flag is set to `true` by default.
+
+This flag indicates whether [`childViewEventPrefix`](./events.md#a-child-views-event-prefix)
+for all views will return the default value of `'childview'` or if it will return `false`
+disabling [automatic event bubbling](./events.md#event-bubbling).
```

Comment:
So if I understand correctly, if we do:

```js
Marionette.setEnabled('childViewEventPrefix', false);

const myView = new Marionette.View();
myView.childViewEventPrefix() === false;

Marionenette.setEnabled('childViewEventPrefix', true);
myView.childViewEventPrefix() === true;
```

I'm just wondering if, when I'm building an app, I'd be tripped up by that flag being changed somewhere I didn't expect (granted it's something that you shouldn't be changing willy-nilly).